tools/count_citation.py

```diff
@@ -1,21 +1,23 @@
 from __future__ import annotations
 
 import json
+import os
 from pathlib import Path
 
 import pandas as pd
 import requests
 from bids_website.utils import data_dir
 from pyzotero import zotero
 from rich import print
-import os
 
 DEBUG = False
 VERBOSE = True
 MINIMUM_YEAR = 2016
 
 # TODO: switch this to a .env file
-os.environ["OPEN_CITATIONS_ACCESS_TOKEN"] = "/home/remi/Documents/tokens/open_citations_access_token.txt"
+os.environ["OPEN_CITATIONS_ACCESS_TOKEN"] = (
+    "/home/remi/Documents/tokens/open_citations_access_token.txt"
+)
```

Comment:
how big is the token -- should we place it into secrets and automate this on CI?

Reply:
in any case -- IMHO better to use some env var and not hardcode paths (or evil yarik would login and steal that exact file!)